runtime/vm/jfr.cpp

```diff
@@ -1020,7 +1021,7 @@ jfrThreadContextSwitchRate(J9VMThread *currentThread)
 				jfrEvent->switchRate = 0;
```

Comment:
Suggest this should avoid the implicit cast:
```c
				jfrEvent->switchRate = 0f;
```

Reply:
`0f` is not a valid floating point literal so have to use `0.0f`.